account/manager.go

```diff
@@ -52,6 +53,8 @@ const (
 	// both extremes for valid account expirations.
 	minAccountExpiry = 144       // One day worth of blocks.
 	maxAccountExpiry = 144 * 365 // A year worth of blocks.
+
+	txLabelPrefixTag = "poold -- %s"
```

Comment:
Is there somewhere better to put this? It isn't specific to `account` although it's only used in relation to accounts right now.

Reply:
I think that's fine. We could export it so we could re-use the constant in Faraday. But maybe that's overkill and might produce a circular package dependency.
The on-chain footprint of Pool will always be related to accounts, so the location is okay too IMO.